Views/VMDisplayMetalViewController.m

```diff
@@ -101,6 +102,13 @@ - (void)viewDidLoad {
     self.mtkView.delegate = _renderer;
     
     [self initTouch];
+    // Pointing device support on iPadOS 13.4 GM or later
+    if (@available(iOS 13.4, *)) {
+        // Betas of iPadOS 13.4 did not include this API, that's why I check if the class exists
+        if (NSClassFromString(@"UIPointerInteraction") != nil) {
```

Comment:
Can you weak-link it?

Reply:
It's part of UIKit, I don't know if one can weak link only parts of it.

Reply:
I think you'd add something like `-Wl,-U,_OBJC_CLASS_$_UIPointerInteraction` to the build flags.